package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "yura-training-camp",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.8",
+    "@fortawesome/free-solid-svg-icons": "^5.5.0",
+    "@fortawesome/react-fontawesome": "^0.1.3",
+    "normalizr": "^3.3.0",
```

Comment:
Not sure how it differs from a https://necolas.github.io/normalize.css/ but it is a different lib.